### DISCUSSION TITLE
So if we aren’t allowed to used private messaging here…

### DISCUSSION THREAD

**Discussion by Sagetvmaster** [OpenSageTV/Community#43] [Q&A]
Is there any alternative that can be arranged?
I’ve always liked that we could discuss things privately regarding things like used equipment, solving a problem you don’t want to clog up the site with, etc.
Another thing about being able to assign email to our old SageTV Forum accounts: we could get sent messages when a thread updated, someone responded, etc.
I haven’t figured out how to get updates sent to me when somebody posts a new message, or answers an old one on the GitHub threads.
Is that possible?

**Comment by wnjj** [comment]
Email updates are done through notifications and subscriptions:

https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github

When I reply, it automatically subscribes.  If you aren't getting emails you may need to check your account settings.

**Reply by Sagetvmaster** [reply]
okay, thanks.  Either no one is leaving many topic messages, or yes, my settings are incorrect.
